core/tests/protractor_desktop/navigation.js

```diff
@@ -130,6 +130,28 @@ describe('Static Pages Tour', function() {
       by.css('.protractor-test-get-started-page')).isPresent()).toBe(true);
   });
 
+  it('should visit the Login page', async function() {
+    await browser.get('/login');
+    await waitFor.pageToFullyLoad();
+    expect(await element(
+      by.css('.protractor-test-login-page')).isPresent()).toBe(true);
```

Comment:
Can you turn this into a waitFor.presenceOf? That will give the element time to load (note that `waitFor.pageToFullyLoad` only waits for the loading message to disappear. The various elements don't always appear instantly once the loading message goes away

Reply:
Done.